errors/error.go

```diff
@@ -0,0 +1,26 @@
+package errors
+
+import "errors"
+
+//Errors for /service/csv.go
```

Comment:
Comments syntax is `// Foo bar baz.`

A space after the`//` and ending with a `.`.

Please fix this in all other project occurrences.

```suggestion
// Errors for /service/csv.go.
```